### ISSUE TITLE
Could not find SC by order id

### ISSUE THREAD

**Issue by dutchwebdesign** [buckaroo-it/Magento2#395]
Hello we have issue in version 1.38.1 with Magento 2.4.2

[2021-09-29 13:43:09] buckaroo_logging.ERROR: 61546d6d9db14|1632922989.6459|Could not find SC by order id:340076518 [] 

While Second Chance is disabled it still wants to remove the SC record, If I check the code this always happens in SuccessOrder.php
```

`    public function execute(\Magento\Framework\Event\Observer $observer)
    {
        $this->logging->addDebug(__METHOD__ . '|1|');

        /* @var $order \Magento\Sales\Model\Order */
        $order = $observer->getEvent()->getOrder();

        try {
            $this->secondChanceRepository->deleteByOrderId($order->getIncrementId());
        } catch (\Exception $e) {
            $this->logging->addError('Could not find SC by order id:' . $order->getIncrementId());
        }`
```

**Comment by vladislav-padalka-hys** [comment]
Good afternoon @dutchwebdesign !

Thanks for your report, but that issue has been eliminated 2 days ago by this commit https://github.com/buckaroo-it/Magento2/commit/3424ee2c887ee991fe38aea195799871e1908393 and will be included into upcoming release

**Comment by dutchwebdesign** [comment]
great thanks do you know when the next release is planned?

**Comment by Buckaroo-Rens** [comment]
Released in 1.39.0
